### PR TITLE
[bitnami/common] Node affinity values must be quoted.

### PR DESCRIPTION
**Pull request by wrossmann**
Label values _must_ be strings, but if they are not quoted then
boolean/numerical/etc values can be misinterpreted by kubernetes.

eg:

```
values:
  - "true"
```
**Description of the change**

Ensure that values in `matchExpressions` for `nodeAffinity` rules are quoted as strings.

**Benefits**

Value strings that can be misinterpreted as non-string types, eg: `"true"`, will no longer be.

**Possible drawbacks**

None

**Applicable issues**

#6349 

**Additional information**

Nada

**Checklist** 
- [X] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/).
- [X] Title of the PR starts with chart name (e.g. `[bitnami/chart]`)

### CODE REVIEW

File: bitnami/common/Chart.yaml [hunk 1]
```diff
@@ -20,4 +20,4 @@ sources:
   - https://github.com/bitnami/charts
   - http://www.bitnami.com/
 type: library
-version: 1.4.3
+version: 1.4.4
```
Comment (juan131) [inline bitnami/common/Chart.yaml:23]: This is a special chart and both `appVersion` and `version` should be bumped

File: bitnami/common/templates/_affinities.tpl [hunk 2]
```diff
@@ -12,7 +12,7 @@ preferredDuringSchedulingIgnoredDuringExecution:
           operator: In
           values:
             {{- range .values }}
-            - {{ . }}
+            - {{ quote . }}
```
Comment (juan131) [inline bitnami/common/templates/_affinities.tpl:15]: You can also use a pipe:
```suggestion
            - {{ . | quote }}

```

**Review by juan131 — COMMENTED** [review]
Thanks so much for this!

I'm okay with your changes, but please address the comments I added.

File: bitnami/common/templates/_affinities.tpl (hunk 2 above)
Comment (wrossmann) [reply bitnami/common/templates/_affinities.tpl:15]: Ahh that was my initial impulse, but I switch between Jinja and Go templates so much I never recall which is which. `{{ . | quote }}` looked "too Jinja-ish" and the Helm docs suggested `{{ quote . }}`.

I'll change it around to be consistent with the other charts and bump the appversion.

**Review by juan131 — APPROVED** [review]
LGTM

**Pull request merged**